lib/src/presentation/widget/pokemons.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import '../../core/util/string_constants.dart';
+
+class Pokemons extends StatelessWidget {
```

Comment:
This class could have a more descriptive name as a widget, like PokemonCard or PokemonPresentationCard